ui/packages/form/src/Dropdown.svelte

```diff
@@ -1,27 +1,139 @@
 <script lang="ts">
-	import { createEventDispatcher } from "svelte";
+    import { fly } from 'svelte/transition';
 	import { BlockTitle } from "@gradio/atoms";
-
+    export let id = '';
+    export let readonly = false;
+    export let placeholder = '';
 	export let label: string;
-	export let value: string | undefined = undefined;
+	export let value: string | Array<string> | undefined = undefined;
+	export let multiselect: boolean = false;
 	export let choices: Array<string>;
 	export let disabled: boolean = false;
 	export let show_label: boolean;
+  
+    let input, 
```

Comment:
There is a weird bug where hitting enter on the typeable field makes the whole app crash

![multiselect_enter](https://user-images.githubusercontent.com/41651716/209021520-a5802ee8-66ca-491f-8800-3c3492928ce7.gif)